packages/default/scss/tooltip/_theme.scss

```diff
@@ -1,80 +1,31 @@
 @mixin kendo-tooltip--theme-base() {
-
-    .k-tooltip-wrapper {
-        .k-tooltip {
-            box-shadow: $kendo-tooltip-shadow;
-        }
-    }
-
-
     // Default tooltip
     .k-tooltip {
         @include fill(
             $kendo-tooltip-text,
             $kendo-tooltip-bg,
             $kendo-tooltip-border
         );
+        
+        box-shadow: $kendo-tooltip-shadow;
```

Comment:
We can maybe use the box-shadow mixin for consistency here e.g. @include box-shadow( $kendo-tooltip-shadow ) ?